<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimControllerClockRate</class>
 <widget class="QFrame" name="SimControllerClockRate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>55</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::Panel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="WkSimController::ClockRateSlider" name="mSlider">
     <property name="minimum">
      <number>-10</number>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="tracking">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="WkSimController::ClockRateSpinBox" name="mSpinBox">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>0</height>
      </size>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="prefix">
      <string>x</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="mMinLabel">
       <property name="font">
        <font>
         <pointsize>6</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Slow</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>12</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="mMaxLabel">
       <property name="font">
        <font>
         <pointsize>6</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Fast</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WkSimController::ClockRateSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>SimControllerToolbarWidgets.hpp</header>
  </customwidget>
  <customwidget>
   <class>WkSimController::ClockRateSlider</class>
   <extends>QSlider</extends>
   <header>SimControllerToolbarWidgets.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
